ci-scripts/dockerfiles/Lmod.dockerfile

```diff
@@ -7,7 +7,9 @@ FROM ubuntu:20.04
 ENV TZ=Europe/Zurich
 ENV DEBIAN_FRONTEND=noninteractive
 ENV _LMOD_VER=8.4.12
-WORKDIR /root
+
+# ReFrame user
```

Comment:
Can you expand the comment there why we need a special user? If it's a standard practice, there is no need.

Reply:
It's just better to have a normal user without root privileges to be as close as possible to the typical use case.